index.rst

```diff
@@ -300,14 +319,10 @@ Full Table of Contents
    setup
    help
    pullrequest
-   runtests
-   coverage
-   docquality
-   documenting
-   silencewarnings
+   testing/index
+   documenting/index
    fixingissues
-   tracker
-   triaging
+   triaging/index
```

Comment:
```suggestion
   issues/index
```

Maybe a little bikesheddy, but since issues (reporting, labels and triaging) is the overarching theme here, the index pages talks about issues in general and there's a sub-page already named triaging, wouldn't it make more sense and be less potentially confusing to call the top-level section `issues` rather than `triaging`?